src/js/site/main.js

```diff
@@ -421,3 +421,21 @@ $(function () {
     });
   });
 });
+
+/*
+  Check if users have dark mode enabled
+  Set key if user response has not already been tracked to ensure we dont get multiple events from the same user
+ */
+$(document).ready(() => {
+  const storageKey = 'provided-dark-mode-reponse';
+  if (localStorage.getItem(storageKey)) {
+    return;
+  } else {
+    localStorage.setItem(storageKey, true);
+    window.AnalyticsClient.trackAction('User Dark Mode Preference', {
+      darkModeEnabled:
+        window.matchMedia &&
+        window.matchMedia('(prefers-color-scheme: dark)').matches,
```

Comment:
I just want to verify that the trackUser allows us to match the analytics_id to a trackAction. in cases where localStorage cannot cover such as when we have users using multiple browsers/machines.

Reply:
analytics_id is User ID in Amplitude, so we can group the data using User ID for an event thats associated with a trackAction to see all their results.